migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MisoServiceManager.java

```diff
@@ -1376,4 +1379,26 @@ private void updatePoolQcDaoDependencies() {
     if (poolDao != null) poolDao.setPoolQcDAO(poolQcDao);
   }
 
+  private void setDefaultSequencingParametersDao() {
+    HibernateSequencingParametersDao dao = new HibernateSequencingParametersDao();
+    dao.setSessionFactory(sessionFactory);
+    dao.setPlatformStore(platformDao);
+    setSequencingParametersDao(dao);
+  }
+
+  public HibernateSequencingParametersDao getSequencingParametersDao() {
+    return sequencingParametersDao;
+  }
+
+  public void setSequencingParametersDao(HibernateSequencingParametersDao sequencingParametersDao) {
+    this.sequencingParametersDao = sequencingParametersDao;
+    updateSequencingParametersDao();
+  }
+
+  private void updateSequencingParametersDao() {
```

Comment:
Another idiomatic change: `setDefaultRunDao()` should contain a line of `dao.setSequencingParametersDao(sequencingParametersDao)`